src/compile.js

```diff
@@ -235,22 +235,49 @@ export class Transformer extends BasisTransformer {
         v0.forEach((v) => {
           if (Number.isInteger(v)) {
             attrs.push(`border-${v}`);
-          } else {
-            attrs.push(v);
-          }
-          if (v == "red") {
+          } else if (v == "slate") {
             attrs.push(`border-${v}-600`);
-          } else if (Number.isInteger(v) >= 100) {
-            attrs.push(`border-red-${v}`)
-          } else {
-            attrs.push(v);
-          }
-          if (v == "sky") {
+          } else if (v == "gray") {
             attrs.push(`border-${v}-600`);
-          } else {
-            attrs.push(v);
-          }
-          if (v == "green") {
+          } else if (v == "zinc") {
```

Comment:
@dyerjesse I see what you are doing here. It is correct, but it doesn't align with what you do in the lexicon. See my comment there.